wp3/wp3_ex6/ex6.c

```diff
@@ -0,0 +1,99 @@
+// (C) Emma Litvin, Nicole Quinstedt. Group: 11 (2023)
+// Work package 3
+// Exercise 6
+// Submission code: XXXX
+
+/*------------------------------------------------------------------------------------------
+Exercise 6: The program reads a string with maximum of 20 characters from the keyboard and
+ stores the string in a local variable.
+ String is copied to another string by using:
+ a) the library function strcpy(),
+ b) custom-made function void copyString() without using any library function.
+ Main program ends by printing out the copied string in the console.
+
+ The program is able to read from both the keyboard and a text file 'myfile.txt' containing
+ one string of characters. The file is created with any file editor.
+ The reading from the text file is done by redirect the reading from command line when the
+ program execution starts.
+-------------------------------------------------------------------------------------------*/
+
+// Include section
+#include <stdio.h>
+#include <string.h>
+
+// Define section
+#define MAX 20                       // Maximum size of the array
+
+FILE *my_file;                       // Pointer to the file
+
+// Function declaration
+void copyString(char original[], char duplicate[]);
+
+// Main function
+int main (int argc, char* args[])
+{
+    // Variable declarations
+    char word[MAX],                     // String that is read from the console
+         copy[MAX];                     // String to copy word string
+
+    // Case if no argument was provided
+    if(args[1] == 0) {
```

Comment:
This works too, but I have noticed that using argc is easier for different conditions when you want to basically do condition on the number of arguments